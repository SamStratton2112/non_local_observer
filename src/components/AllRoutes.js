import { Routes, Route, Navigate } from 'react-router-dom';
import Contact from './Contact';




const AllRoutes = () =>{
    return(
        <Routes >
            {/* <Route exact key="" path="/" element={</>}/> */}
            {/* <Route exact key="about" path="/about" element={<About/>}/> */}
            <Route exact key="contact" path="/contact" element={<Contact/>}/>
            {/* <Route exact key="cart" path="/cart" element={<Cart/>}/> */}
            {/* <Route exact key="shop" path="/shop" element={<Shop/>}/> */}
            {/* <Route exact key="home" path="/" element={<Home/>}/> */}
            <Route path='*' element={<Navigate to='/'/>}/>
        </Routes>
    )
};

export default AllRoutes;