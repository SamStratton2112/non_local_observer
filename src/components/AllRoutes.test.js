import AllRoutes from './AllRoutes';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';

describe(AllRoutes, () => {
    it("renders without crashing", () => {
        render(<MemoryRouter>
            <AllRoutes />
        </MemoryRouter>);
    });
    it("matches snapshot", () => {
        const { asFragment } = render(<MemoryRouter>
            <AllRoutes />
        </MemoryRouter>);
        expect(asFragment()).toMatchSnapshot();
    });
});