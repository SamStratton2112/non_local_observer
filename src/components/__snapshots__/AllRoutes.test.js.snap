// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AllRoutes matches snapshot 1`] = `<DocumentFragment />`;
